src/g_game.c

```diff
@@ -579,7 +579,7 @@ void G_PrepTiccmd(void)
 
 void G_BuildTiccmd(ticcmd_t* cmd)
 {
-  const boolean strafe = M_InputGameActive(input_strafe);
+  const boolean strafe = M_InputGameActive(input_strafe) && !basecmd.angleturn;
```

Comment:
Does this issue exist in Woof 12.0.2? We don't seem to have checked this before: https://github.com/fabiangreffrath/woof/blob/f75ebb283f8318f8ed54f1448d1808fce01fb18c/src/g_game.c#L385

Reply:
No, this is unique to Woof 14.0.0 and 14.1.0 because mouse/gamepad turning is decoupled now.